src/request/components/RequestDetailPanelData.tsx

```diff
@@ -0,0 +1,48 @@
+import { MasterDetailTriPanel } from './MasterDetailTriPanel';
+import { RequestDetailPanelDataOperationCommandContainer } from '../containers/RequestDetailPanelDataOperationCommandContainer';
+import { RequestDetailPanelDataOperationTableContainer } from '../containers/RequestDetailPanelDataOperationTableContainer';
+
+import * as React from 'react';
+import Highlight = require('react-highlight');
+
+export interface IDataComponentProps {
+    totalOperationCount: number;
+}
+
+export class DataComponent extends React.Component<IDataComponentProps, {}> {
+    public render() {
+        return (
+            <div className='tab-data'>
+            {
+                this.props.totalOperationCount > 0
+                    ? <MasterDetailTriPanel masterPanel={this.renderMaster()} leftDetailPanel={this.renderLeftDetail()} leftDetailPanelTitle='Command' rightDetailPanel={this.renderRightDetail()} rightDetailPanelTitle='Summary' />
+                    : <div className='tab-section text-minor'>Could not find any data.</div>  
+            }
+            </div>
+        );
+    }
+    
+    private renderMaster() {
+        return (
+            <div className='tab-data-master'>
+                <div className='tab-data-operation-count'>{this.getTotalOperationCountText()}</div>
+                <br />
+                <RequestDetailPanelDataOperationTableContainer />
+            </div>
+        );
+    }
+    
+    private renderLeftDetail() {
+        return (
+            <RequestDetailPanelDataOperationCommandContainer />
+        )
+    }
+    
+    private renderRightDetail() {
+        return null;
+    }
+    
+    private getTotalOperationCountText() {
+        return this.props.totalOperationCount + (this.props.totalOperationCount === 1 ? ' Operation' : ' Operations');
+    }
```

Comment:
What this would probably look like is:

```javascript
function pluralize(testValue: number, oneFormatString: string, manyFormatString: string, zeroFormatString: string, formatArgs?: string[]);
```
Example:

```javascript
const oneOperation = '%n Operation';
const zeroOrManyOperations = '%n Operations';
<div>{pluralize(count, oneOperation, zeroOrManyOperations, zeroOrManyOperations, [ count ])}</div>
```
You pass in the test value, then the (optionally formatted) resource strings for each case, then (optionally) the argument list for any formatted strings.  If no argument list was passed, then maybe you could assume that the test value is the implied first argument (if the strings were formatted).  That would allow you to drop the args list in most simple cases.